soweego/target_selection/musicbrainz/common.py

```diff
@@ -0,0 +1,21 @@
+import os
+import requests
+
+def get_path():
+    """Return the path of the current module"""
+    path = os.path.abspath(__file__)
+    return os.path.dirname(path)
+
+def get_output_path():
+    """Returns the path of the output files"""
+    output_dir_path = '%s/output' % get_path()
+
+    if not os.path.exists(output_dir_path):
+        os.makedirs(output_dir_path)
+
+    return output_dir_path
+
+def api_request_wikidata(query):
```

Comment:
This should be moved to the `wikidata` package if not already implemented, otherwise just deleted.